source/demo-code/recipe-info-rough/recipe-page.css

```diff
@@ -393,3 +394,41 @@ li.item {
 li.steps {
   margin-top: 0.5rem;
 }
+
+/* Style for timer */
+div.timer {
+  align-items: center;
+  background-color: white;
+  border: 3px solid gold;
+  border-radius: 25px;
+  bottom: 0;
+  display: block;
+  float: left;
+  height: 140px;
+  left: 0;
+  margin: 1rem;
+  overflow: auto;
+  padding: 1rem;
+  position: fixed;
+  width: auto;
```

Comment:
maybe nice to also add`cursor: grab;` on hover

Reply:
Added